go.mod

```diff
@@ -154,7 +156,7 @@ require (
 	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/tetratelabs/wazero v1.2.1 // indirect
 	github.com/vbatts/tar-split v0.11.3 // indirect
-	github.com/vladimirvivien/gexe v0.2.0 // indirect
+	github.com/vladimirvivien/gexe v0.2.0
```

Comment:
this should be moved to the require stanza above with all the other direct dependencies

Reply:
Not sure, but it was doing the right thing for me when i removed them and just run `go mod vendor`